src/System.Private.CoreLib/shared/Interop/Windows/Kernel32/Interop.GetSystemTimes.cs

```diff
@@ -0,0 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal partial class Kernel32
+    {
+        [DllImport(Libraries.Kernel32, SetLastError = true)]
+        internal extern static bool GetSystemTimes(out long idle, out long kernel, out long user);
```

Comment:
>out long idle, out long kernel, out long user [](start = 51, length = 45)

Should the parameters be of [FILETIME](https://docs.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.comtypes.filetime?view=netstandard-2.0) type?